src/Core/Event.js

```diff
@@ -0,0 +1,45 @@
+/**
+ * @type {Object}
+ */
+var Event = {
+	/**
+	 * @param {Element} element
+	 * @param {string} type
+	 */
+	addEventListener: function addEventListener (element, type) {
+		if (!this[type]) {
+			this[type] = new Map()
```

Comment:
Map not supported in IE <= 10, needs shim

Reply:
That's a mistake should be WeakMap, there's a shim for that un `Utility.js` called `Hash`

Reply:
Updated.